tests/phpunit/test-class-wp-widget-image.php

```diff
@@ -0,0 +1,468 @@
+<?php
+/**
+ * Unit tests covering WP_Widget_Image functionality.
+ *
+ * @package    WordPress
+ * @subpackage widgets
+ */
+
+/**
+ * Test wp-includes/widgets/class-wp-widget-image.php
+ *
+ * @group widgets
+ */
+class Test_WP_Widget_Image extends WP_UnitTestCase {
+
+	/**
+	 * Test get_instance_schema method.
+	 *
+	 * @covers WP_Widget_Image::get_instance_schema
+	 */
+	function test_get_instance_schema() {
+		if ( version_compare( PHP_VERSION, '5.3', '<' ) ) {
+			return $this->markTestSkipped( 'ReflectionMethod::setAccessible is only available for PHP 5.3+' );
+		}
+
+		$wp_widget_image     = new ReflectionClass( 'WP_Widget_Image' );
+		$get_instance_schema = $wp_widget_image->getMethod( 'get_instance_schema' );
+		$get_instance_schema->setAccessible( true );
+
+		$schema = $get_instance_schema->invoke( new WP_Widget_Image );
```

Comment:
Conventions are to include parens when invoking classes, so I think this should be `new WP_Widget_Image()`.

This is something we should improve in WPCS.